kong/api/routes/upstreams.lua

```diff
@@ -107,4 +109,54 @@ return {
       crud.post(self.params, dao_factory.targets)
     end,
   },
+
+  ["/upstreams/:name_or_id/targets/active/"] = {
```

Comment:
@Tieske, what do you think  the chances of #2252 getting in before this? Can you merge this PR in and then rebase your change? Or shall this change wait until #2252 is in, and then we'll rebase?